Rafael Gonzalez/Ejercicio2.py

```diff
@@ -0,0 +1,13 @@
+numero = int(input("Elige el tamaño de la piramide: "))
+for i in range(numero+1):
+    print ("A"*i) 
+
+for i in range(numero+i):

```

Comment:
Es malo usar la i del for pasado aqui, deberías de guardarla de alguna manera y llamarle diferente.
Porque la i pertenece al for anterior y esta i es una nueva variable.
Funciona el código, peeeeero, no es tan correcto lo que está escrito